htmd/parameterization/esp.py

```diff
@@ -201,24 +210,27 @@ def __init__(self):
 
         self._reciprocal_distances = None
 
+        self.equivalent_atom_groups = None
+        self.equivalent_group_by_atom = None
+        self.netcharge = None
```

Comment:
This is wrong! The charge has to come from `qm_result` object.

Reply:
You will need to help me with this. I only see some `mulliken` charges in `QMResult`. What should I do with them?